libraries/rush-lib/src/api/RushConfigurationProject.ts

```diff
@@ -184,6 +185,11 @@ export class RushConfigurationProject {
    */
   public readonly tags: ReadonlySet<string>;
 
+  /**
+   * If this project is in a subspace, and which one
+   */
+  public readonly subspace?: string;
```

Comment:
```suggestion
  public readonly subspaceName?: string;
```

Reply:
Also  `public readonly subspaceName: string | undefined;` seems like a more natural notation.  

Practically they are equivalent, but conceptually this example is "possibly entirely missing":

```ts
function makeReport(options: { verbose?: string});
```

...whereas this example is "always present but sometimes assigned to `undefined`":
```ts
class ReportConfig {
  public verbose: string | undefined;
  
  public constructor() { this.verbose = undefined; }
}
```

Reply:
Changed to `public readonly subspaceName: string | undefined;`